.github/workflows/release.yml

```
@@ -0,0 +1,92 @@
+name: Publish Python distributions to PyPI
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+env:
+  DATAPATH: "/usr/share/cobbler"
+  DOCPATH: "/usr/share/man"
+  ETCPATH: "/etc/cobbler"
+  LIBPATH: "/var/lib/cobbler"
+  LOGPATH: "/var/log"
+  COMPLETION_PATH: "/usr/share/bash-completion/completions"
+  STATEPATH: "/tmp/cobbler_settings/devinstall"
+
+  HTTPD_SERVICE: "apache2.service"
+  WEBROOT: "/var/www";
+  WEBCONFIG: "/etc/apache2/vhosts.d";
+  WEBROOTCONFIG: "/etc/apache2";
+  TFTPROOT: "/srv/tftpboot"
+  DEFAULTPATH: "etc/sysconfig"
+
+  APACHE_USER: "www-data"
+  HTTP_USER: "www-data"
+  APACHE_GROUP: "www-data"
+
+jobs:
+  build-n-publish:
+    name: Build and publish Python distributions to PyPI and TestPyPI
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@master
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
```

```suggestion
      - uses: actions/checkout@v2
      - name: Set up Python 3.9
        uses: actions/setup-python@v2
        with:
          python-version: 3.9
```
Use a more recent version of Python, [setup-python V2](https://github.com/actions/setup-python) and [checkout V2](https://github.com/actions/checkout).